apps/condo/domains/organization/integrations/sbbol/sync/requestTransactions.js

```diff
@@ -0,0 +1,184 @@
+const dayjs = require('dayjs')
+const { get, isEmpty } = require('lodash')
+const { validate: uuidValidate } = require('uuid')
+
+const { getLogger } = require('@open-condo/keystone/logging')
+const { getSchemaCtx } = require('@open-condo/keystone/schema')
+
+const { BANK_INTEGRATION_IDS } = require('@condo/domains/banking/constants')
+const { BankAccount, BankTransaction, BankIntegration, BankIntegrationContext } = require('@condo/domains/banking/utils/serverSchema')
+const { ISO_CODES } = require('@condo/domains/common/constants/currencies')
+const { dvSenderFields, INVALID_DATE_RECEIVED_MESSAGE } = require('@condo/domains/organization/integrations/sbbol/constants')
+const { SBBOL_IMPORT_NAME } = require('@condo/domains/organization/integrations/sbbol/constants')
+const { initSbbolFintechApi } = require('@condo/domains/organization/integrations/sbbol/SbbolFintechApi')
+
+const { ERROR_PASSED_DATE_IN_THE_FUTURE } = require('../constants')
+
+
+const logger = getLogger('sbbol/SbbolSyncTransactions')
+const isTransactionsReceived = (response) => (get(response, 'error.cause', '') !== 'STATEMENT_RESPONSE_PROCESSING')
+
+/**
+ * Connects new BankTransaction records for BankAccount according to transaction data from SBBOL.
+ *  @param {String} userId
+ *  @param {BankAccount[]} bankAccounts
+ *  @param {keystoneContext} context
+ *  @param {String} statementDate
+ *  @param {String} organizationId
+ */
+async function _requestTransactions ({ userId, bankAccounts, context, statementDate, organizationId }) {
```

Comment:
I created this function to make the code easier to read. In fact, it is not needed and everything can be described in one function, but this way the logic is visible more clearly. I think since it is not imported, it is more correct to leave it with `_` in the name so that it would be clear that this is a private function exclusively for use inside this module

Reply:
The proposed name better represents meaning. You have higher-level function, that enumerates dates in given interval and helper-function that does the job for specified date.